### PR TITLE
Create fed recruitment page

### LINKED ISSUE #2463: Draft and finalize content for feds-only recruitment page
The draft  of the [feds-only recruitment page](https://docs.google.com/document/d/1UoaxVKe9iKwUvH7U26Xgx6l5PZR2ZHRsFHf0t-i0hvk/edit?usp=sharing) content is nearly ready to be finalized and published on the site and linked from the refreshed research page.

### PR DESCRIPTION
**Pull request by jaclinec**
# Summary

Created new page for federal employee recruitment for usability research. The page shares information on the purpose of our research, what to expect, and how to sign up.

## Related issue

Closes #2463 

## Preview link

[Preview link](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jaclinec-patch-1//about/research/recruitment/general-fed/)

## Problem statement

We need an official and public-facing way to communicate how USWDS federal users can participate in usability research with us and offer them a way to sign up. No page currently exists.

## Solution

Create a new recruitment page for federal USWDS users that is linked from the main research page.

## Major changes

_For complex PRs, create a list of the significant updates made._

## Testing and review

- [ ] Information is accurate
- [ ] [Recruitment for Feds](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jaclinec-patch-1/about/research/recruitment/general-fed/) is accessible.
- [ ] Research has three links in sidenav: General public, Feds, and Latest updates. 
- [ ] Changelog exists at the bottom of the page.
- [ ] Zero a11y issues.

### CODE REVIEW

File: pages/about/research/recruitment/general-fed.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+category: About
+lead: Help shape the future of U.S. government websites.
+redirect_from:
+  - /research/recruitment/federal/
+changelog:
+  key: about-research-recruitment-federal
+subnav: true
+---
+
+How the federal USWDS community can participate in our research
```
Comment (finekatie) [inline pages/about/research/recruitment/general-fed.md:14]: Please remove this line. It's duplicating the page title.

File: pages/about/research/recruitment/general-fed.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**To sign up:**  Fill out a brief research signup form or email us at <{{ site.uswds_email }}> to tell us you’re interested. We’ll reach out when we have a research opportunity for you.
+
+Your participation is completely voluntary and you can opt out at any time by emailing us at <{{ site.uswds_email }}>.
+
+## What to expect if we invite you to participate
+
+{:.border-top-2px .border-base-lighter .padding-top-1}
+
+We’ll share the research purpose and details with you when we reach out to see if you’re interested in participating. Typically, participating could mean:
```
Comment (finekatie) [inline pages/about/research/recruitment/general-fed.md:33]: Recommend changing "to see" to: "to find out" or "to learn". The alternate phrases offered here are more a11y-friendly.

File: pages/about/research/recruitment/general-fed.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+- Offering asynchronous feedback on a concept
+- Having a 30-60 minute conversation with us
+- Testing a prototype and thinking out loud (which we would ask to record)
```
Comment (finekatie) [inline pages/about/research/recruitment/general-fed.md:37]: Wondering how this phrasing might affect non-verbal participants. Just a thought.

**Comment by amycole501** [comment]
1. Seems to be a run-on sentence in the first paragraph. Should it read:
How the federal USWDS community can participate in our research. 

Getting feedback from designers, developers, UX practitioners, content strategists, and others in the USWDS community is critical for us to deliver a design system that works for you.

1 a. Also, I'd recommend changing that above sentence to read:
We are actively seeking feedback from federal employees including designers, developers, UX practitioners, content strategies, and others. Getting input from the USWDS community is critical for us to deliver a design system that works for you._

2.  Are we missing the signup form link? I didn't see it.

**Comment by jaclinec** [comment]
Here is the [content doc for the fed recruitment page for reference.](https://docs.google.com/document/d/1UoaxVKe9iKwUvH7U26Xgx6l5PZR2ZHRsFHf0t-i0hvk/edit?usp=sharing).

**Review by finekatie — COMMENTED** [review]
Looking good! Only one other issue/question: Under "What to expect..", do you want to mention anything about what happens after they've participated? Like will we share anything with them or can they follow up on GH to learn about the work?

**Comment by jaclinec** [comment]
> 1. Seems to be a run-on sentence in the first paragraph. Should it read:
>    How the federal USWDS community can participate in our research.
> 
> Getting feedback from designers, developers, UX practitioners, content strategists, and others in the USWDS community is critical for us to deliver a design system that works for you.
> 
> 1 a. Also, I'd recommend changing that above sentence to read: We are actively seeking feedback from federal employees including designers, developers, UX practitioners, content strategies, and others. Getting input from the USWDS community is critical for us to deliver a design system that works for you._
> 
> 2. Are we missing the signup form link? I didn't see it.

Yes, let's delete that first sentence - its a dupe. I also like Amy's suggested edit to make it clearer that we're actively seeking participants.

It is missing the signup form links as well as the 18F Privacy Page link - it's in the content doc I linked above for reference.

Finally, I think we should replace the subtitle of "Help shape the future of U.S. government websites." to "Help shape the future of USWDS."

File: pages/about/research/recruitment/general-fed.md (hunk 2 above)
Comment (amycole501) [reply pages/about/research/recruitment/general-fed.md:33]: People who are blind often use "see" so I think it's fine to use.

File: pages/about/research/recruitment/general-fed.md (hunk 1 above)
Comment (mejiaj) [reply pages/about/research/recruitment/general-fed.md]: Removed in eb51406. Thanks!

**Comment by mejiaj** [comment]
Thanks all, added comments in eb51406.

**Updates**
- Removed duplicate sentence
- Re-worded first paragraph
- Attached touchpoints form
- Attached 18F privacy link

**Review by finekatie — APPROVED** [review]
Looks good to me!

**Comment by jaclinec** [comment]
Yup, looks great! Only one tiny thing - please bold the sentence in the 'sign up' area that says "We’re looking for federal government employees who either currently use USWDS or who are curious about USWDS."

Other than that I think it's good to go!

**Comment by mejiaj** [comment]
@jaclinec added your request to bold sentence in 7129daf.

**Comment by mejiaj** [comment]
@annepetersen this is ready for your review

**Review by annepetersen — APPROVED** [review]
LGTM!

**Comment by annepetersen** [comment]
Noticed this after the fact: can we correct the capitalization in the side-nav item? Should be "Recruitment for feds" rather than "Recruitment for Feds" @mejiaj

**Pull request merged**